doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -952,7 +952,7 @@ The commands to fine-tune the reduction strategies and the lazy conversion
 algorithm are described in this section.  Also see :ref:`Args_effect_on_unfolding`,
 which supports additional fine-tuning.
 
-.. cmd:: Opaque {+ @reference }
+.. cmd:: Opaque {? ! } {+ @reference }
```

Comment:
How about adding a sentence below that explains what the `!` does?  Let's not force readers to guess what it means--or worse, to not use the feature because they don't about it.  As I like quip, "undocumented features don't exist".

Reply:
Will do once we decide what to do with this new syntax (it is really meant for debugging only).